TVShows/TVShows/Modules/Login/LoginInterfaces.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  LoginInterfaces.swift
+//  TVShows
+//
+//  Created by Ivan Milicevic on 18/10/2018.
+//  Copyright (c) 2018 Ivan Milicevic. All rights reserved.
+//
+//  This file was generated by the 🐍 VIPER generator
+//
+
+import UIKit
+import Alamofire
+
+enum LoginNavigationOption {
+    case home
+}
+
+protocol LoginWireframeInterface: WireframeInterface {
+    func navigate(to option: LoginNavigationOption)
+}
+
+protocol LoginViewInterface: ViewInterface {
+    func setRememberState(state: Bool)
```

Comment:
Better naming would be:

```swift
func setRememberState(_ state: Bool)
```
just to avoid named first argument since you already named it in `RememberState`